src/rpcnet.cpp

```diff
@@ -165,6 +173,18 @@ UniValue getpeerinfo(const UniValue& params, bool fHelp)
         }
         obj.push_back(Pair("whitelisted", stats.fWhitelisted));
 
+        UniValue sendPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapSendBytesPerCmd) {
+            sendPerCmd.push_back(Pair(i.first, i.second));
+        }
+        obj.push_back(Pair("bytessent_per_cmd", sendPerCmd));
+
+        UniValue recvPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapRecvBytesPerCmd) {
+            recvPerCmd.push_back(Pair(i.first, i.second));
```

Comment:
Why would this result in more controllable parsing? The zeros don't mean anything. The only reason those entries are there in the first place is our 'ignore unknown command' implementation detail.